linux_os/guide/system/permissions/files/file_permissions_unauthorized_sgid/rule.yml

```diff
@@ -8,6 +8,11 @@ description: |-
     unauthorized SGID files is determine if any were not installed as part of an
     RPM package, which is cryptographically verified. Investigate the origin
     of any unpackaged SGID files.
+    This configuration check whitelists SGID files which were installed via RPM.
```

Comment:
I suggest changing 'whitelists' to 'considers authorized'.
Pleaes, do the same change in the other rule description.